atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,241 @@
+            best_medoid_idx = int(np.argmax(similarity_score))
+            properties = { "properties": [{
+                "competency_id": competency_id,
+                "name" : competency["properties"]["name"],
+                "text": competency["properties"]["text"],
+                "cluster_id": clusters[best_medoid_idx]["properties"]["cluster_id"]
+            }]}
+            self.weaviate_client.add_embeddings(CollectionNames.COMPETENCY.value, embedding, properties)
+
+        for index in range(len(texts)):
+            text_entry = texts[index]
+            competency_id = self.weaviate_client.get_embeddings_by_property(CollectionNames.COMPETENCY.value, "cluster_id", labels[index])["properties"]["competency_id"]
```

Ruff (0.11.9)

Line too long (169 > 88)

(E501)